manifests/openshift/prometheus-rule.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    tier: node
+    wasp.io: ""
+  name: wasp-rules
+  namespace: openshift-monitoring
```

Comment:
@enp0s3 @Barakmor1 In openshift, in order for Prometheus to collect this PrometheusRule we need to choose one of the following:
- Deploy it in `openshift-monitoring` namespace, as I'm currently doing.
- Deploy it in wasp namespace, and then we need to add `openshift.io/cluster-monitoring: "true"` to that namespace. 

Please let me know if you think the approach I chose is problematic. 

**Note:** In CI I'm deploying it in wasp namespace without the need for additional adjustments, and that's enough because of https://github.com/openshift-virtualization/wasp-agent/blob/main/cluster-up/cluster/k8s-provider-common.sh#L163-L167.